src/lib.rs

```diff
@@ -307,7 +307,6 @@ where
                     initial_count += 1;
                     let mut internals = self.0.internals.lock().await;
                     if initial_count == initial_size {
-                        debug!("initial ok");
```

Comment:
Actually, it used to be 'println!'. 
https://github.com/importcjj/mobc/blob/6c8c4a007c2a6fa1fcbe4205f42628b9e9397cb9/src/lib.rs#L310
After I released 0.2.9, I found this problem and changed it to 'debug!', and it hasn't been released yet. Don't worry, I'll release v0.2.10 immediately.

Reply:
Hah, ok :)

Reply:
Thought this is a bug in the `select!` macro.

Reply:
v0.2.10 released